
{'date': 'Fri Jul 03 2020 11:43:49.552 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 7
    np.mean(jogadores)
AttributeError: 'module' object has no attribute 'mean'
'''},
{'date': 'Thu Jul 30 2020 16:47:09.51 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 76
    Kwarwp(jogo)
NameError: name 'jogo' is not defined
'''},
{'date': 'Thu Jul 30 2020 17:19:00.310 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 98
    Kwarwp(Jogo, mapa=MAPA_ROCHA)
TypeError: __init__() got an unexpected keyword argument 'mapa'
'''},
{'date': 'Thu Jul 30 2020 17:25:26.492 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 73
    Kwarwp(Jogo)
  module <module> line 51
    self.cena = self.cria(mapa=mapa) if vitollino else None
TypeError: cria() got an unexpected keyword argument 'mapa'
'''},
{'date': 'Thu Jul 30 2020 17:26:17.691 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 73
    Kwarwp(Jogo)
  module <module> line 50
    self.cena = self.cria(mapa=mapa) if vitollino else None
TypeError: cria() got an unexpected keyword argument 'mapa'
'''},
{'date': 'Thu Jul 30 2020 17:26:27.120 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 73
    Kwarwp(Jogo)
  module <module> line 50
    self.cena = self.cria(mapa=mapa) if vitollino else None
TypeError: cria() got an unexpected keyword argument 'mapa'
'''},
{'date': 'Thu Jul 30 2020 17:26:28.8 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 73
    Kwarwp(Jogo)
  module <module> line 50
    self.cena = self.cria(mapa=mapa) if vitollino else None
TypeError: cria() got an unexpected keyword argument 'mapa'
'''},
{'date': 'Thu Jul 30 2020 17:26:28.475 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 73
    Kwarwp(Jogo)
  module <module> line 50
    self.cena = self.cria(mapa=mapa) if vitollino else None
TypeError: cria() got an unexpected keyword argument 'mapa'
'''},
{'date': 'Thu Aug 20 2020 16:09:31.187 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 215
  fabrica = {
                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Aug 20 2020 16:57:56.431 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 295
    Kwarwp(Jogo, mapa=MAPA_INICIO)
  module <module> line 199
    self.cena = self.cria(mapa=self.mapa) if vitollino else None
  module <module> line 233
    self.taba = {(i, j): fabrica[imagem].objeto(
  module <module> line 249
    coisa = Indio(imagem, x=0, y=0, cena=cena, taba=self)
  module <module> line 116
    self.azimute = self.AZIMUTE.n
AttributeError: 'Indio' object has no attribute 'AZIMUTE'
'''},
{'date': 'Thu Aug 20 2020 17:01:06.792 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 126
  self.indio.siz = (lado*3, lado*4)
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Aug 20 2020 17:02:53.503 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 127
  self.indio.siz = (lado*3, lado*4)
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Aug 20 2020 17:02:54.620 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 127
  self.indio.siz = (lado*3, lado*4)
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Aug 20 2020 17:58:39.963 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 112
    class Indio():
  module <module> line 114
    AZIMUTE = Rosa(Ponto(0, -1),Ponto(1, 0),Ponto(0, 1),Ponto(-1, 0),)
NameError: name 'Rosa' is not defined
'''},
{'date': 'Thu Aug 20 2020 17:58:58.307 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 127
  self.indio.siz = (lado*3, lado*4)
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Aug 20 2020 17:59:07.294 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 112
    class Indio():
  module <module> line 114
    AZIMUTE = Rosa(Ponto(0, -1),Ponto(1, 0),Ponto(0, 1),Ponto(-1, 0),)
NameError: name 'Rosa' is not defined
'''},
{'date': 'Thu Aug 20 2020 17:59:17.28 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 127
  self.indio.siz = (lado*3, lado*4)
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Sep 03 2020 15:31:51.159 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 127
  self.indio.siz = (lado*3, lado*4)
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Sep 03 2020 15:35:46.86 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 296
    class Kwarwp():
  module <module> line 310
    def __init__(self, vitollino=None, mapa = MAPA_INICIAL, medidas = {}):
NameError: name 'MAPA_INICIAL' is not defined
'''},
{'date': 'Thu Sep 03 2020 15:38:05.811 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 32
  """
                                      ^
SyntaxError: invalid syntax
'''},
{'date': 'Thu Sep 03 2020 16:16:16.131 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 276
  def limpa(self):
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Sep 03 2020 16:21:53.790 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 281
  def limpa(self):
  ^
IndentationError: expected an indented block
'''},